tests/functional/iac/test_iac_scan_all.py

```diff
@@ -8,22 +8,68 @@
 from tests.repository import Repository
 
 
-def test_iac_scan_all(tmp_path: Path) -> None:
+# `iac scan` is set for deprecation, but should behave exactly as `iac scan all` in the meantime
+pytestmark = pytest.mark.parametrize("scan_prefix", [["all"], []])
```

Comment:
It seems very hacky to me to use a list and then unfold it to handle the case where no command is given.

I think you should define a helper to build the args that will handle explicitely the case where there is no command.